standalone-metastore/metastore-server/pom.xml

```diff
@@ -204,6 +204,11 @@
       <artifactId>hive-storage-api</artifactId>
       <version>${storage-api.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-serde</artifactId>
```

Comment:
@kgyrtkirk Is this change okay? I mean we could have used reflection again to call serde classes but it will make code more complex and non-readable.

Reply:
`TypeInfo` and friends in the metastore...I don't think we want to do it :)
I don't yet have a concreate idea how to avoid it - because in case we want to evaluate partition values correctly we would need the same stuff we use on the ql side...

technically this dependency means that if we want to move out the metastore - we should also take the serde stuff with it...

Reply:
Fixed!